### ISSUE TITLE
Bug: Google Meet can't share screen

### ISSUE THREAD

**Issue by ysoudy** [ferdium/ferdium-app#744]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.2.1-nightly.27

### What Operating System are you using?

Windows

### Operating System Version

Windows 11

### What arch are you using?

x64

### Expected Behavior

Hi,

Google Meet service still working, but we can't share our screen anymore.
It was working last week

### Actual Behavior

An error popup : Votre navigateur n'est pas compatible avec le partage d'écran
In english : Your browser is not compatible with screen sharing

### Steps to reproduce

1. Create a service with Google Meet
2. Join a meeting
3. Start to share your screen

### Screenshots

![image](https://user-images.githubusercontent.com/3369324/200306693-35751566-60fd-4d61-91ed-63ad55c8b072.png)

**Comment by cino** [comment]
Confirmed, having the same issue while "Screen Recording" is enabled on MacOS for Ferdium.

**Comment by ysoudy** [comment]
any other feedback or fix ? ;)

**Comment by baw-benji** [comment]
I don't know if it worked before, but I can't share on Linux (Debian) either.

**Comment by dap39** [comment]
Did you try to change Ferdium's user agent to Chrome's latest?

**Comment by baw-benji** [comment]
> Did you try to change Ferdium's user agent to Chrome's latest?

Good point. I changed the user agent to the one in Chrome (which is working), but still the same result.
Btw. Microsoft Teams is able to share.

**Comment by dkweva01007** [comment]
all google meet's features are disabled

**Comment by NP-compete** [comment]
Any update on this?

**Comment by tosmi** [comment]
same here, meet works but no screen sharing under fedora with gnome.
